app/data/serializers.py

```diff
@@ -0,0 +1,32 @@
+from django.contrib.auth.models import User, Group
+from rest_framework.authtoken.models import Token
+from rest_framework import serializers
+
+
+class UserSerializer(serializers.HyperlinkedModelSerializer):
+    class Meta:
+        model = User
+        fields = ('id', 'url', 'username', 'email', 'groups', 'password',
+                  'is_staff', 'is_superuser',)
+        read_only_fields = ('id',)
+        write_only_fields = ('password',)
+
+    def create(self, validated_data):
```

Comment:
That makes sense. We can configure the OpenID setup to create new users on first login separately.